R/serialization.R

```diff
@@ -0,0 +1,42 @@
+# This file will be autogenerated.
+
+#' Serialize/Unserialize an mlpack model.
```

Comment:
```suggestion
#' Serialize/deserialize an mlpack model.
```
:)

Reply:
Sadly, _R_ calls this "unserialize" ☹️

Reply:
No problem; best to stick with the R conventions. :+1: